crates/store/src/db/rocksdb.rs

```diff
@@ -37,11 +37,7 @@ impl Database for RocksDB {
     }
 
     fn has(&self, col: Column, key: Slice) -> eyre::Result<bool> {
-        let cf_handle = self.try_cf_handle(&col)?;
-
-        let exists = self.db.key_may_exist_cf(cf_handle, key.as_ref());
-
-        Ok(exists)
+        self.get(col, key).map(|value| value.is_some())
```

Comment:
make this `self.db.key_may_exist_cf(..) && self.get(..).map(..)`

since we know for certain when it doesn't exist, but are not quite sure if it does

so we only check to be sure when it may exist